express-post-json/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+
+const app = express();
+
+const gradesArray = [];
+let nextId = 1;
+let grades = {};
+
+app.get('/api/grades', (req, res) => {
+  gradesArray.push(grades);
+  res.json(gradesArray);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades = req.body;
```

Comment:
Instead of replacing the `grades` object, add this new one as a property, like the data model from `express-get-json`